src/index.ts

```diff
@@ -1,7 +1,5 @@
-import { useLocalStorage } from './use-localstorage';
+export { useLocalStorage } from './use-localstorage';
```

Comment:
If you change this to `export { useLocalStorage }`, then `useLocalStorage` no longer exists in this file. The `export default useLocalStorage;` here doesn't work anymore